### PR TITLE
fix empty content for sportiva.shueisha.co.jp

### LINKED ISSUE #177378: sportiva.shueisha.co.jp
### Issue URL (Annoyance)
 [https://sportiva.shueisha.co.jp/clm/football/wfootball/2024/04/12/35/](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fsportiva.shueisha.co.jp%2Fclm%2Ffootball%2Fwfootball%2F2024%2F04%2F12%2F35%2F)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>
<details>
  <summary>Screenshot 2</summary>

  Screenshot 2
</details>
<details>
  <summary>Screenshot 3</summary>

  Screenshot 3
</details>

### System configuration

Information | Value
--- | ---
AdGuard product: | AdGuard Browser Extension v4.3.46
System version: | Windows11
Browser: | Chrome
Stealth mode: | disabled
Filters: | <b>Ad Blocking:</b><br/>AdGuard Base<br/><br/><b>Privacy:</b><br/>AdGuard Tracking Protection, <br/>AdGuard URL Tracking<br/><br/><b>Social Widgets:</b><br/>AdGuard Social Media<br/><br/><b>Annoyances:</b><br/>AdGuard Annoyances<br/><br/><b>Language-specific:</b><br/>AdGuard Japanese

### PR DESCRIPTION
**Pull request by acediat**
fix: #177378

### CODE REVIEW

File: BaseFilter/sections/specific.txt [hunk 1]
```diff
@@ -344,6 +344,7 @@ $xmlhttprequest,third-party,denyallow=rsc.cdn77.org|linkvertise.com|fastly.net|s
 ! NOTE: Specific rules
 !
 ||ezojs.com/ezoic/
+shueisha.co.jp##ul[class="list-basic-a c-box list-basic-a--align-center u--padding-top-20 u--height-min-250 u--hidden-if-mobile"]
```
Comment (jellizaveta) [inline BaseFilter/sections/specific.txt:347]: `shueisha.co.jp##.list-basic-a:has(> li > div.c-ad)`

**Review by jellizaveta — CHANGES_REQUESTED** [review]

**Review by jellizaveta — APPROVED** [review]

File: BaseFilter/sections/specific.txt [hunk 2]
```diff
@@ -344,8 +344,7 @@ $xmlhttprequest,third-party,denyallow=rsc.cdn77.org|linkvertise.com|fastly.net|s
 ! NOTE: Specific rules
 !
 ||ezojs.com/ezoic/
-shueisha.co.jp##ul[class="list-basic-a c-box list-basic-a--align-center u--padding-top-20 u--height-min-250 u--hidden-if-mobile"]
-cbs12.com##div[id^="js-openWebConversation-Container"]
+shueisha.co.jp##.list-basic-a:has(> li > div.c-ad)
```
Comment (BlazDT) [inline BaseFilter/sections/specific.txt:347]: Move to Japanese filter -> Specific.

File: SpywareFilter/sections/allowlist.txt [hunk 3]
```diff
@@ -1228,10 +1228,14 @@ goodwillfinds.com#%#//scriptlet("set-constant", "_etmc", "emptyArr")
...
-@@||csm.cxpublic.com/Shueisha.js$domain=wpb.shueisha.co.jp
-@@||scdn.cxense.com/cx.cce.js$domain=wpb.shueisha.co.jp
+@@||csm.cxpublic.com/Shueisha.js$domain=wpb.shueisha.co.jp|sportiva.shueisha.co.jp
+@@||scdn.cxense.com/cx.cce.js$domain=wpb.shueisha.co.jp|sportiva.shueisha.co.jp
 @@||cxense.com/cx.js$domain=shueisha.co.jp
 @@||api.cxense.com/public/widget/data?json=$domain=shueisha.co.jp
+! https://github.com/AdguardTeam/AdguardFilters/issues/177378
+@@||googletagmanager.com/gtm.js$domain=sportiva.shueisha.co.jp
+@@||csm.cxpublic.com/Shueisha/domain/sportiva.shueisha.co.jp.js$domain=sportiva.shueisha.co.jp
+@@||cdn.cookielaw.org/scripttemplates/otSDKStub.js$domain=sportiva.shueisha.co.jp
```
Comment (BlazDT) [inline SpywareFilter/sections/allowlist.txt:1238]: Spyware filter is not blocking cookielaw (or is it showing up for you in filtering log when cookie filter is disabled?

**Review by BlazDT — CHANGES_REQUESTED** [review]

File: SpywareFilter/sections/allowlist.txt (hunk 3 above)
Comment (jellizaveta) [reply SpywareFilter/sections/allowlist.txt:1238]: most likely duplicated

**Review by jellizaveta — CHANGES_REQUESTED** [review]

**Review by BlazDT — APPROVED** [review]

**Review by jellizaveta — APPROVED** [review]

**Pull request merged**